### ISSUE TITLE
IllegalArgumentException if glue path contains space

### ISSUE THREAD

**Issue by bsideup** [cucumber/cucumber-jvm#903]
```
Exception in thread "main" java.lang.IllegalArgumentException: Not a file or directory: /Users/bsideup/Google
	at cucumber.runtime.io.FileResourceIterator$FileIterator.<init>(FileResourceIterator.java:54)
	at cucumber.runtime.io.FileResourceIterator.<init>(FileResourceIterator.java:20)
	at cucumber.runtime.io.FileResourceIterable.iterator(FileResourceIterable.java:19)
	at cucumber.runtime.groovy.GroovyBackend.loadGlue(GroovyBackend.java:75)
	at cucumber.runtime.Runtime.<init>(Runtime.java:91)
	at cucumber.runtime.Runtime.<init>(Runtime.java:69)
	at cucumber.runtime.Runtime.<init>(Runtime.java:65)
	at cucumber.api.cli.Main.run(Main.java:35)
	at cucumber.api.cli.Main.main(Main.java:18)
```

My glue path is "/Users/bsideup/Google Drive/Work/cool-project/src/test/groovy/"

Thanks!

**Comment by aslakhellesoy** [comment]
This was fixed in [1.2.3](https://github.com/cucumber/cucumber-jvm/blob/master/History.md#123-2015-07-07), so I'm assuming you're using an older version. If that's not the case, please ask to reopen this and let us know what version you're on.

**Comment by folaAkin** [comment]
Please reopen this issue,I am using version 1.2.4
